autometa/common/external/samtools.py

```diff
@@ -26,58 +26,65 @@
 import logging
 import os
 import subprocess
+import shutil
+import tempfile
 
 import pandas as pd
 import multiprocessing as mp
 
 logger = logging.getLogger(__name__)
 
 
-def run(cmd):
-    """Run `cmd` via subprocess.
-
+def run(cmd, bam):
```

Comment:
If you're incorporating \`bam\` into the run function, you could probably just incorporate `run` into `sort`.  Since no other functions exist in the script that would call run. This may simplify things further